lib/skate_web/router.ex

```diff
@@ -26,13 +26,13 @@ defmodule SkateWeb.Router do
   scope "/", SkateWeb do
     pipe_through [:redirect_prod_http, :browser]
 
-    get "/", PageController, :index
+    get("/", PageController, :index)
   end
 
   scope "/api", SkateWeb do
-    pipe_through :api
+    pipe_through(:api)
```

Comment:
Why does this get `()`, but the `pipe_through` above doesn't? Was this automatic formatting?